projects/conda-forge-simulation-stack.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Packaging the HEP simulation stack on conda-forge
+postdate: 2024-02-08
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+skillset:
+  - Python
+  - C++
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - ATLAS
+  - LHCb
+  - CMS
+  - ALICE
+  - EIC
+  - HLLHC
+program:
+  - IRIS-HEP fellow
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Package the HEP simulation tools for conda-forge
+description: >
+  One common toolchain used in high energy physics for simulation is:
+  [MadGraph5_aMC@NLO](https://launchpad.net/mg5amcnlo) to
+  [PYTHIA8](https://pythia.org/) to [Delphes](https://github.com/delphes/delphes).
+  Installing these tools can be challenging at times, especially for new users.
+  [Conda-forge](https://conda-forge.org/) allows for distribution of arbitrarily
+  complex binaries across multiple platforms though the conda/mamba/micromamba/pixi
+  package management ecosystem.
+  As [ROOT](https://anaconda.org/conda-forge/root/) has been successfully packaged and
+  distributed on conda-forge along with the
+  [PYTHIA8 Python bindings](https://anaconda.org/conda-forge/pythia8/) it should be
+  possible to package all the components of the HEP simulation stack and distribute
+  them on conda-forge.
+
+  However, the interconnected nature of some of these tools requires that multiple
+  dependencies are first packaged and distributed before the full stack can be.
+  This project would attempt to package as many of the dependencies of the HEP
+  simulation stack on conda-forge as possible starting with
+  [FastJet](http://fastjet.fr/), [LHAPDF](https://lhapdf.hepforge.org/),
+  and adding Python 3 bindings for the
+  [HepMC2](https://github.com/conda-forge/hepmc2-feedstock), and
```

Comment:
Ah. @chrisburr, @agbuckley, @20DM from looking at https://gitlab.cern.ch/hepmc/HepMC I see now that there never was Python bindings for HepMC2 at all and that they were only added in HepMC3. If that's correct and I haven't missed something, then that means that this bit of the proposal is a "no go" as HepMC2 development has long since stopped and there's no point in trying to add Python bindings to it, and we can close PR https://github.com/conda-forge/hepmc2-feedstock/pull/4.

Reply:
Yep. There were a couple of different external packages providing bindings, e.g. I had made a SWIG-based one many years before, but nothing official/built-in until very recently.

Reply:
Thanks @agbuckley. I didn't do my research well enough on this before starting things off :grimacing:, and I'm happily (but embarrassingly) finding that multiple projects described in this proposal already have Python bindings on conda-forge

* HepMC3: https://github.com/conda-forge/pyhepmc-feedstock
* LHAPDF: https://github.com/conda-forge/lhapdf-feedstock

or won't need them in the case of HepMC2.